pandas/tests/test_algos.py

```diff
@@ -1412,6 +1412,20 @@ def test_value_counts_uint64(self):
 
         tm.assert_series_equal(result, expected)
 
+    def test_value_counts_series(self):
+        # GH#54857
+        values = np.array([3, 1, 2, 3, 4, np.nan])
+        with tm.assert_produces_warning(FutureWarning, match="deprecated"):
+            result = pd.value_counts(values, bins=3)
```

Comment:
Can we just test the `pd.Series(values).values_counts(bins=3)` version instead?

Reply:
weirdly enough, that one did not raise the error